pkg/arvo/sys/hoon.hoon

```diff
@@ -10595,12 +10596,16 @@
       ==  ==  ==
     ::
     ++  dext
+      =<  $
+      ~%  %nest-dext  +  ~
+      |.
       ^-  ?
       =-  ?:  -  &
           ?.  tel  |
           ::  ~_  (dunk %need)
           ::  ~_  (dunk(sut ref) %have)
-          ~>(%mean.[%leaf "nest-fail"] !!)
+          ~_  [%leaf "nest-fail"]
```

Comment:
There are a few other `%mean` hints likes this for other compiler for error messages (think `%mull-bunk-*`). If this was invalid, they will be as well. It doesn't strictly need to be resolved right now, but it's worth noting.